babel/spelling.py

```diff
@@ -0,0 +1,549 @@
+# -*- coding: utf-8 -*-
```

Comment:
@erickwilder hi, I think I already updated the commit, but maybe it was not succesful. `number_spelling.py` is only a rename of `spelling.py` suggested by @sils1297